amnesia/src/XCompileToWindows.toolchain

```diff
@@ -1,26 +1,29 @@
-# the name of the target operating system
+# The name of the target operating system
 set(CMAKE_SYSTEM_NAME Windows)
 
-# which compilers to use for C and C++
-# set(CMAKE_C_COMPILER   x86_64-w64-mingw32-gcc-win32)  # 64 bit but the windows version was made for 32 only
-# set(CMAKE_CXX_COMPILER x86_64-w64-mingw32-g++-win32)  # 64 bit
-set(CMAKE_C_COMPILER   i686-w64-mingw32-gcc-win32)
-set(CMAKE_CXX_COMPILER i686-w64-mingw32-g++-win32)
+# Which compilers to use for C and C++
+# Uncomment these lines for 64-bit compilation, though these are set for a 32-bit Windows environment
+# set(CMAKE_C_COMPILER   x86_64-w64-mingw32-gcc-win32)  # 64-bit GCC compiler, targeting 32-bit Windows
```

Comment:
Redundant to say that this is GCC/G++ (it's in the compiler name). Also I don't think it's targeting 32 bit if it's explicitly a 64 bit compiler?

Reply:
Removed.